extensions/amp-carousel/0.1/scrollable-carousel.js

```diff
@@ -79,6 +84,14 @@ export class AmpScrollableCarousel extends BaseCarousel {
     this.container_.addEventListener(
         'scroll', this.scrollHandler_.bind(this));
 
+    this.registerAction('goToSlide', invocation => {
+      const args = invocation.args;
+      if (args) {
+        const index = parseInt(args['index'], 10);
+        this.goToSlide_(index);
+      }
+    }, ActionTrust.HIGH);
```

Comment:
Is high trust needed?

Reply:
Done.

Reply:
Actually I think the reason this is high trust is because we allow amp-bind for the `[slide]` attribute, which also must update when change the slide (no matter how). I reverted the trust change on `type="slides"`. We can keep it low trust on `type="carousel"` until we implement the `slideChange` event and other support for `amp-bind`.